docs/deployment.md

```diff
@@ -0,0 +1,75 @@
+# Deploying Find - Manage Courses App - Step by Step Guide 
+
+All members of the Find development team are able to deploy into any of the environments.
+
+## 1. Check what you're deploying
+
+Go to the Find Build pipelines [Frontend](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_build?definitionId=29) & [Backend](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_build?definitionId=46)  and identify the commit you want to deploy. It is advisable to make note of the build id at this stage. 
+
+Make sure to check the diff on GitHub to see if there's anything risky.
+
+You also have to make sure that you're deploying only work that has been product reviewed and approved for deployment. 
+See "To Deploy" colum on the  [Find Team board](https://trello.com/b/fXA6ioZN/team-board-find-team)
+The "Product Review" column on the [Find Team board](https://trello.com/b/fXA6ioZN/team-board-find-team) should be empty.
+
+## 2. Tell the team ![](https://www.webfx.com/tools/emoji-cheat-sheet/graphics/emojis/loudspeaker.png)
+
+Summarise what you're deploying and tell the team in Slack on the `#twd_find` channel. Use ![](https://www.webfx.com/tools/emoji-cheat-sheet/graphics/emojis/rotating_light.png)  `:alert:`  and  ![](https://www.webfx.com/tools/emoji-cheat-sheet/graphics/emojis/loudspeaker.png)`:loudspeaker:` as required.
+
+
+## 3. Deploy to staging
+
+1. Load the [Find Postgraduate Teacher Training release pipeline](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_release?_a=releases&view=mine&definitionId=36)  in Azure DevOps.
+2. Click the blue `:rocket:` "Create Release" button  at the top right of the page which will open "Create a new release" sub-menu. 
+3. Release is made up of several stages i.e. backend, frontend, integration test. Deselect the stages that you wish to run manually by changing its trigger from automated to manual (optional).
+4. Under Artifacts section, master branch build artifacts should be already selected from the drop down list by default. Should you wish to change this select the version for the artifact sources for `manage-courses-backend`  and `manage-courses-frontend` drop down list.
+5. Specify release description (optional).
```

Comment:
Same thing about prefixing, although more for consistency at this point as the sentence is a lot shorter.

Reply:
fixed